### PR TITLE
Web on android update

### LINKED ISSUE #5508: Web on Android [6/30/2021]
* [Draft](https://docs.google.com/document/d/1VTufPRiqf7NPXMzC4mRBt0A3Npkwo-rhf8mf51F9yTA/edit?usp=sharing)

### PR DESCRIPTION
**Pull request by andreban**
Fixes #5508 

Changes proposed in this pull request:

- Adds security considerations for using WebView as an in-app browser.

### CODE REVIEW

File: src/site/content/en/blog/web-on-android/index.md [hunk 1]
```diff
@@ -60,6 +60,45 @@ web platform features, developers would still need to write code and implement t
...
+available in modern browsers.
+
+##### Multi-process architecture and site isolation
+
+Browsers are designed to be secure while rendering and executing content that is untrusted. To
+ensure the user stays safe while navigating content that is potentially untrustworthy or even
+malicious, modern browsers employ techniques such as using [multi-process architecture][9] and
+[site isolation][10].
+
+Without the multi-process architecture, a crash caused by the web page can crash the entire browserapp, or
```
Comment (jpmedley) [inline src/site/content/en/blog/web-on-android/index.md:78]:
```suggestion
Without the multi-process architecture, a crash caused by the web page can crash the entire browser app, or
```

File: src/site/content/en/blog/web-on-android/index.md [hunk 2]
```diff
@@ -60,6 +60,45 @@ web platform features, developers would still need to write code and implement t
...
+render content that they own and trust, but can be a problem for applications running untrusted
+third-party content, like in-app browsers, and leaves users exposed to vulnerabilities like
+[Meltdown][11] and [Spectre][12], which could be used for stealing cookies, banking details,
+personal information, and more.
+
+##### Secure UI Indicators
+
+It is also important to provide good security indicators to users, and browsers put a lot of effort
+and are [always evolving][13] in this area. However, the WebView lacks an API for checking if a
+site's connection is secure, which allows application developers to build trustworthy security
```
Comment (jpmedley) [inline src/site/content/en/blog/web-on-android/index.md:98]:
```suggestion
site's connection is secure that allows application developers to build trustworthy security
```

**Review by jpmedley — CHANGES_REQUESTED** [review]
Sorry I didn't catch these in the Google Doc.

**Comment by andreban** [comment]
> Sorry I didn't catch these in the Google Doc.

No problem, thanks for checking them out. I added the changes.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**